docs-conceptual/azps-9.2.0/configure-global-settings.md

```diff
@@ -0,0 +1,96 @@
+---
+description: Configure Azure PowerShell global settings using centralized and granular AzConfig cmdlets
+ms.custom: devx-track-azurepowershell
+ms.date: 01/09/2023
+ms.devlang: powershell
+ms.service: azure-powershell
+ms.topic: conceptual
+title: Configure Azure PowerShell global settings
+---
+
+# Configure Azure PowerShell global settings
+
+Azure PowerShell supports several global settings like disabling breaking change warning
+messages. Prior to Az PowerShell module version 9, there wasn't a centralized or granular way to
+configure global settings for Azure PowerShell.
```

Comment:
This may be a good place to explain the benefits of having global settings? This would say "why" customers want to use Az Config.